.editorconfig

```diff
@@ -10,3 +10,6 @@ insert_final_newline = true
 charset = utf-8
 indent_style = tab
 indent_size = 4
+
+[*.yaml]
+indent_size = 2
```

Comment:
while we are at it, also add `indent_style = spaces` _[see why](https://yaml.org/faq.html)_

Reply:
> nice solution! have you tried it out in another branch/repo already?

Yes, extensively. Take a look at https://github.com/helpwave/helpwave-maxrs-services *(cloned from helpwave/services into my personal account, moved to our organization for this pr. Gets deleted when this pr gets merged.)*